WordPress/Classes/Utility/PingHubManager.swift

```diff
@@ -10,7 +10,6 @@ private func defaultAccountToken() -> String? {
         return nil
     }
     guard let token = account.authToken, !token.isEmpty else {
-        assertionFailure("Can't create a PingHub client if the account has no auth token")
```

Comment:
Should there be an error log message here, to replace the assertionFailure?

Reply:
No, after reviewing it I believe I was wrong to add an assertion there in the first place. PingHubManager is designed to deal with a logged out state. It can function without a default account, so it should work without a token, and it's observing the `WPAccountDefaultWordPressComAccountChanged` notification, which we now post when the token changes for the default account, so it should recover fine after login (although I realize now that I haven't tested this scenario)

Reply:
good to know +1

Reply:
I tested this, and it seems we're only posting `WPAccountDefaultWordPressComAccountChanged` when we clear the token, but not when we log back in. I'm OK with the bug for now, since it's not really related to the changes, and I think the impact is quite low (you'd still get push notifications if enabled, pull-to-refresh notifications would work, and pinghub would be working again after a relaunch)